src/Controllers/AuthCodeController.php

```diff
@@ -111,6 +112,7 @@ public function startAuthorizationGrant(array $scopes, ?string $email=null): str
             'response_type' => 'code',
             'scope'         => join(' ', $scopes),
             'email'         => $email,
+            'user_id'       => $accessToken,
```

Comment:
Should this query string parameter be conditionally added, if `$accessToken` is not null?

Reply:
Seeing these changes are used in https://github.com/npr/www-render/pull/1806

Reply:
Yeah that makes sense. I was following the existing pattern of how "$email" is handled, though.

Reply:
Can this be an access token, and not an actual user ID?

Reply:
@todo Jared to get back to me on this on how to best send access token, or instead to use user model

Reply:
If there is code already in www-render that gets a user model from an anonymous Id, then can use that to send along here